src/murfey/util/models.py

```diff
@@ -17,6 +18,7 @@ def __repr__(self) -> str:
             "Visit("
             f"start='{self.start:%Y-%m-%d %H:%M}', "
             f"end='{self.end:%Y-%m-%d %H:%M}', "
+            f"session_id='{self.session_id!r}',"
```

Comment:
```suggestion
            f"session_id='{self.session_id}',"
```
don't need the `repr()` call when we know it's an integer.